packages/platform-core/src/usage-data/usage_data_emitter.test.ts

```diff
@@ -124,7 +124,7 @@ void describe('UsageDataEmitter', () => {
    */
   const setupAndInvokeUsageEmitter = async (testData: {
     isSuccess: boolean;
-    error?: AmplifyError;
+    error?: AmplifyError<string>;
```

Comment:
Nit: don't need to specify string since it's the default

Reply:
It was not default, but now it is.